src/routes/api/decks/user/+server.ts

```diff
@@ -6,11 +6,17 @@ export const GET = (async (event) => {
   const { uid } = await authenticate(event);
   const omitOrgDecks = event.url.searchParams.get('omitOrgDecks') ?? false;
   const orgIdParam = event.url.searchParams.get('orgId');
+
+  const headers = {
+    'Cache-Control': event.request.headers.get('x-cache-nonce') ? 'private, max-age=300, must-revalidate' : 'private, no-cache',
```

Comment:
Translation (and [link](https://csswizardry.com/2019/03/cache-control-for-civilians/) to an all-time bookmark for me)

`private` -> only store this on a user's client, not any intermediate caches
`max-age=300` -> Data is stale after 5 minutes
`must-revalidate` -> When the data is stale, you **have** to fetch it again, you cannot serve it from the cache, even if you are offline
`no-cache` -> technically this does not mean it can't be stored in a cache, but it means you need to revalidate it every single time, and since we don't currently return `304`s from our APIs, that means it will be fetched as usual